modules/search/instant-search/components/search-result.jsx

```diff
@@ -0,0 +1,42 @@
+/** @jsx h */
+
+/**
+ * External dependencies
+ */
+import { sprintf, _n } from '@wordpress/i18n';
+import { h, Component } from 'preact';
+import strip from 'strip';
+
+class SearchResult extends Component {
+	render() {
+		return (
+			<div className="jetpack-instant-search__result">
+				<a
+					href={ `//${ this.props.result.fields[ 'permalink.url.raw' ] }` }
+					target="_blank"
+					rel="noopener noreferrer"
+					className="jetpack-instant-search__result-title"
+				>
+					{ strip( this.props.result.fields.title_html ) || 'Unknown Title' }
```

Comment:
I seem to have encoding issues here:

![image](https://user-images.githubusercontent.com/426388/63436782-98563300-c429-11e9-83b8-8ccf13e7fbfc.png)

Reply:
Ya i've seen this... i think it has to do with the indexing...

Reply:
This is now being tracked in #13291.